wix/Lang/OpenJDK.hotspot.de-de.wxl.template

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file and others below where deleted and added, not moved by git
Don't we lose git history ?

Reply:
I am using `git mv` but I am not sure why those files show up as deleted and added...

Reply:
this commit `https://github.com/AdoptOpenJDK/openjdk-installer/pull/252/commits/d657ca2aaa5391c5606476cc06d481f748230741` is rename, this commit is `https://github.com/AdoptOpenJDK/openjdk-installer/pull/252/commits/80a6eab8df56d6518835ecdda3177d79b3e061fd` update the file. but putting together make the file look like deleted and added :(

Reply:
I am running into the issue this guy had before: https://dev.to/nickytonline/renaming-a-file-via-git-after-it-has-been-deleted-in-a-pr-41dn

Reply:
do you want me to create 2 PRs? the first one renaming files, and the second one adds parameters and updates the cmd file.

Reply:
For the rename / move don't worry, it's all good.
git log --follow wix/Lang/OpenJDK.Base.de-de.wxl.template : OK
git blame : OK
and 
git log --name-status
show rename with 77% similarity
https://stackoverflow.com/questions/53056942/git-diff-name-status-what-does-r100-mean

>commit 5e46b26fb91e39736050ef702f833fc84da98128
>Author: Junyuan Zheng <junyuanz@microsoft.com>
>Date:   Wed Oct 7 10:31:34 2020 -0700
>
>    rename Lang/*
>
>R077    wix/Lang/OpenJDK.Base.de-de.wxl wix/Lang/OpenJDK.Base.de-de.wxl.template
>R077    wix/Lang/OpenJDK.Base.en-us.wxl wix/Lang/OpenJDK.Base.en-us.wxl.template
>R077    wix/Lang/OpenJDK.Base.es-es.wxl wix/Lang/OpenJDK.Base.es-es.wxl.template
>R077    wix/Lang/OpenJDK.Base.fr-fr.wxl wix/Lang/OpenJDK.Base.fr-fr.wxl.template
>R077    wix/Lang/OpenJDK.Base.ja-jp.wxl wix/Lang/OpenJDK.Base.ja-jp.wxl.template
>R076    wix/Lang/OpenJDK.Base.zh-cn.wxl wix/Lang/OpenJDK.Base.zh-cn.wxl.template
>R076    wix/Lang/OpenJDK.Base.zh-tw.wxl wix/Lang/OpenJDK.Base.zh-tw.wxl.template